src/features/wallet/TwoKeyContractModal.tsx

```diff
@@ -70,6 +111,35 @@ const _TwoKeyContractModal: ForwardRefRenderFunction<
             </div>
           </>
         )}
+        {success ? <div>Successfully added two-key contract.</div> : null}
+        {showForm ? (
```

Comment:
small nit, but if you want to show an element if true, and nothing if false i typically do this because its slightly shorter:
` success && <div>Successfully added two-key contract.</div>`

Reply:
Yup, no issue doing it this way.